### PR TITLE
Favourite tokens in selector

### PR DESCRIPTION
**Pull request by W3stside**
# Summary
Part of #574 

### When > 12 favourited tokens in web view
image

### When default or < 12 tokens favourited (note no blur, no extra padding)
image

### Tooltip
image

### Mobile view unaffected
image

## QA
1. head to `/account/tokens`
2. add a bunch of tokens
3. > 12 
4. check web view token selector
5. check mob view token selector
6. be happy (hopefully)

### CODE REVIEW

File: src/custom/state/user/reducer.ts [hunk 1]
```diff
@@ -237,12 +252,8 @@ export default createReducer(initialState, (builder) =>
         delete state.favouriteTokens[chainId][address]
       }
     })
-    // MOD - to remove all favourite tokens
+    // TODO: move to mod
```
Comment (W3stside) [inline src/custom/state/user/reducer.ts:255]:
```suggestion
```

File: src/custom/state/user/reducer.ts [hunk 2]
```diff
@@ -220,15 +238,12 @@ export default createReducer(initialState, (builder) =>
       state.URLWarningVisible = !state.URLWarningVisible
     })
     // MOD - to add/remove favourite token based on if its already added or not
+    // TODO: move to mod
```
Comment (W3stside) [inline src/custom/state/user/reducer.ts:241]:
```suggestion
```

File: src/state/user/reducer.ts [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import { createReducer } from '@reduxjs/toolkit'
+// TODO: properly mod
```
Comment (W3stside) [inline src/state/user/reducer.ts:2]:
```suggestion
```

**Review by anxolin — APPROVED** [review]
Only comment, i would personally prefer that we replace this button by (restore defaults)

Reason is, the defaults are not that many, so its easy to restore and un-star the few if u really want to remove them all. Also, because makes it easier to restore the most common ones, and is coherent with the initial state and behaviour.

Other than this nitpick, I really like it

image

**Pull request merged**